client/src/plugins/update-checks/UpdateChecksAPI.js

```diff
@@ -15,6 +15,8 @@ const OS_INFO_CONFIG_KEY = 'os.info';
 
 const UPDATE_CHECK_API = 'update-check';
 
+const PRODUCT_CAMUNDA_MODELER = 'Camunda Modeler';
```

Comment:
I'd love the product name to be set on build time. Thus, we won't have to change anything in the code when we implement this feature in Zeebe Modeler.

Reply:
👍

Reply:
I think we're ready to merge the PR when you address this.

Reply:
On the other hand, we use hard-coded values in other places: https://github.com/camunda/camunda-modeler/blob/c1b8a6d9e132e16d1f10d7e7c66709d2ef17d6f0/app/lib/index.js#L52

Meh.

Still, 
```suggestion
const PRODUCT_NAME = 'Camunda Modeler';
```
would make it more resilient.

Reply:
I'm ok with injecting this: https://github.com/camunda/camunda-modeler/pull/1855/commits/3e666500a2f0bdbf74db7deee1e64ef66769e79a

Should we keep this commit?

Reply:
ah yeah, let me update the var name

Reply:
Done -> https://github.com/camunda/camunda-modeler/pull/1855/commits/149955b72047fa09f9b0fae46621f7efd914cb0a